docs/pubs.md

```diff
@@ -7,14 +7,14 @@ subtitle:
 ## Papers
 * **Drum: A Rhythmic Approach to Interactive Analytics on Large Data**, Jianfeng Jia, Chen Li, Michael J. Carey.
 [[PDF (Long version)](../img/query-slicing-long.pdf)][[Video](https://youtu.be/gztxneqSIDE)]
-* **Caching Geospatial Objects in Web Browsers**, Taewoo Kim, Vidhyasagar Thirumaraiselvan, Jianfeng Jia, Chen Li, ACM SIGSPATIAL 2017
+* **Caching Geospatial Objects in Web Browsers (Demo Paper)**, Taewoo Kim, Vidhyasagar Thirumaraiselvan, Jianfeng Jia, Chen Li, ACM SIGSPATIAL 2017
```

Comment:
"Demo Paper" is not part of the title, I think.  If so, move it to the end of the line.

Reply:
done